controllers/binding/snapshotenvironmentbinding_adapter.go

```diff
@@ -112,6 +112,11 @@ func (a *Adapter) EnsureIntegrationTestPipelineForScenarioExists() (controller.O
 				a.logger.Error(err, "Failed to create pipelineRun for snapshot, environment and scenario")
 				return controller.RequeueWithError(err)
 			}
+
+			// measure only non-reruns
+			if _, ok := gitops.GetIntegrationTestRunLabelValue(a.snapshot); !ok {
+				metrics.RegisterPipelineRunWithEphemeralEnvStarted(a.snapshot.CreationTimestamp, metav1.Now())
+			}
 			metrics.RegisterPipelineRunWithEphemeralEnvStarted(a.snapshot.CreationTimestamp, metav1.Now())
```

Comment:
Should it be removed since it is called above?

Reply:
ah yes, stupid me

Reply:
fixed